app/services/labor/ping_admins.rb

```diff
@@ -0,0 +1,26 @@
+module Labor
```

Comment:
Placing a service inside `app/services/labor` brings a confusion: there are `app/services`, `app/labor` and `app/services/labor`. I suggest removing the `Labor` namespace and putting this class inside `app/services` or `app/labor`

Reply:
I vote for `app/services` for now